lib/Dialect/TritonGPU/Transforms/Utility.h

```diff
@@ -0,0 +1,12 @@
+#ifndef TRITON_LIB_DIALECT_TRITONGPU_TRANSFORMS_UTILITY_H_
+#define TRITON_LIB_DIALECT_TRITONGPU_TRANSFORMS_UTILITY_H_
+#include "mlir/IR/Matchers.h"
+#include "mlir/IR/PatternMatch.h"
+
+namespace mlir {
+
+void addFixupLoopPattern(MLIRContext *ctx, RewritePatternSet &patterns);
```

Comment:
I think it would be even nicer to have a utility that just fixup loops in a given module, i.e. `fixupLoops(Module op)`.We don't want a full-blown pass because the pass manager should be able to assume that every single pass takes in and output valid IR

Reply:
Done.